wallet-android/app/src/main/cpp/vendors/download.sh

```diff
@@ -0,0 +1,15 @@
+#! /bin/bash
```

Comment:
I had some problems running this script. Some weird line endings. I solved it using `dos2unix` on it.